.github/workflows/functional_test.yaml

```diff
@@ -3,7 +3,7 @@ name: Splunk Platform Functional Test
 on: pull_request
 
 jobs:
-  e2e-test:
+  functional-test:
```

Comment:
Nit: I honestly think we should rename the functional tests to e2e where ever possible. It is one of the odd things this project has compared to others.

Reply:
I am following the feedback from @rmfitzpatrick that these are not truly e2e tests since they don't involve the o11y product, so renaming to functional tests seems to be closer to what they do, plus they replace existing functional tests. No strong opinion about it.